### PR TITLE
[REG-648] Update hardhat@^2.11.2 - post-Merge test

### PR DESCRIPTION
**Pull request by aquiladev**

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -102,6 +102,7 @@ module.exports = {
     hardhat: {
       blockGasLimit: 10000000,
       initialBaseFeePerGas: 0,
+      hardfork: 'merge',
```
Comment (nickshatilo) [inline hardhat.config.js:105]: does that affect sandbox? not sure if it makes sense to rebuild it

**Review by nickshatilo — APPROVED** [review]
lgtm, just one question

File: hardhat.config.js (hunk 1 above)
Comment (aquiladev) [reply hardhat.config.js:105]: it is not, but I think it is good to change hardfork for sandbox and rebuild as well

Comment (aquiladev) [reply hardhat.config.js:105]: just checked, ganache does not support the hardfork

**Pull request merged**